docs/examples/sentence_classification.md

```diff
@@ -0,0 +1,344 @@
+    13.4% of the dataset. The goal is to predict whether an SMS is a spam or not.
+    
+          Name  SMSSpam                                                                              
+          Task  Binary classification                                                                
+       Samples  5,574                                                                                
+      Features  1                                                                                    
+        Sparse  False                                                                                
+          Path  /Users/home/river_data/SMSSpam/SMSSpamCollection                                     
+           URL  https://archive.ics.uci.edu/ml/machine-learning-databases/00228/smsspamcollection.zip
+          Size  466.71 KB                                                                            
+    Downloaded  True                                                                                 
+
+
+
+
+```python
+from pprint import pprint
+
+X_y = datasets.SMSSpam()
+
+for x, y in X_y:
+    
+    pprint(x)
+    print(f'Spam: {y}')
+    break
+```
+
+    {'body': 'Go until jurong point, crazy.. Available only in bugis n great world '
+             'la e buffet... Cine there got amore wat...\n'}
+    Spam: False
+
+
+Let's start by building a simple model like a naive bayes classifier. We will first transform the sentences with a tf-idf to feed our model. Then, we will measure the accuracy of our model with the AUC metric. This metric is adapted when the classes of our dataset are not uniformly distributed. In addition, the Naive Bayes models can perform very well on unbalanced datasets and supports multiple classes for multiple classification problems.
+
+
+```python
+from river import feature_extraction
+from river import naive_bayes
+from river import metrics
+
+def extract_body(x):
+    """Extract the body of the sms."""
+    return x['body']
+
+X_y = datasets.SMSSpam()
+
+model = extract_body | feature_extraction.TFIDF() | naive_bayes.BernoulliNB(alpha = 0)
+
+metric = metrics.ROCAUC()
+
+cm = metrics.ConfusionMatrix()
+
+for x, y in X_y:
+    
+    y_pred = model.predict_one(x)
+    
+    if y_pred is not None:
+        
+        metric.update(y_pred=y_pred, y_true=y)
+        
+        cm.update(y_pred=y_pred, y_true=y)
+    
+    model.learn_one(x, y)
+    
+metric
+```
+
+
+
+
+    ROCAUC: 0.929296
+
+
+
+The confusion matrix:
+
+
+```python
+cm
+```
+
+
+
+
+               False    True
+       False    4809      17
+        True     103     644
+
+
+
+The results are quite good with this first model.
+
+Since we are working with an unbalanced dataset, we can use the `imblearn` module to artificially increase the number of spam in our dataset. For more information about the `imblearn` module, you can find a dedicated tutorial [here](https://riverml.xyz/latest/examples/imbalanced-learning/).
```

Comment:
You shouldn't use a hard link. You should reference relative to where you are. So in this case it would be something like `[here](./imbalanced-learning)`. I'm not exactly sure so I'll let you try locally.